ci/image.travis

```diff
@@ -81,6 +81,8 @@ function vic_script {
     $DRIVER_EXE -v
     $DRIVER_EXE -o
 
+    export OMP_NUM_THREADS=4
```

Comment:
Add a comment (I know this is only for the test run, but it would be good to explain what this).